setup.py

```diff
@@ -77,12 +77,18 @@
     sys.modules['distutils.command.build_ext'].Extension = _Extension
 
 with_cython = False
+if 'sdist' in sys.argv:
+    # we need cython here
+    with_cython = True
 try:
     from Cython.Distutils.extension import Extension as _Extension
     from Cython.Distutils import build_ext as _build_ext
     with_cython = True
 except ImportError:
-    pass
+    if with_cython:
+        raise
+    else:
+        pass
```

Comment:
```python
else:
    pass
```
is a fancy noop, can just remove those two lines

Reply:
true, that looks fancy ;-)
will fix and force-push